README.md

```diff
@@ -450,34 +450,27 @@ To see more of the `react-native-camera` in action, you can check out the source
 
 ## Q & A
 
-#### meta-data android 26	
+#### Q: When I try to build my project, I get following error:
 ```
 AndroidManifest.xml:25:13-35 Error:
      Attribute meta-data#android.support.VERSION@value value=(26.0.2) from [com.android.support:exifinterface:26.0.2] Android
 Manifest.xml:25:13-35
   is also present at [com.android.support:support-v4:26.0.1] AndroidManifest.xml:28:13-35 value=(26.0.1).
         Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:23:9-25:38 to override.
 ```
+#### A: As the error message hints `com.android.support:exifinterface:26.0.2` is already found in `com.android.support:support-v4:26.0.1`
+To fix this issue, modify your project's `android/app/build.gradle` as follows:
+```Gradle
+dependencies {
+    compile (project(':react-native-camera')) {
+        exclude group: "com.android.support"
+    }
 
-Add this to your AndroidManifest.xml:
```

Comment:
can we keep this lines? as well

Reply:
I'm not sure if they are necessary as they didn't really fix the issue, at least for me.
The issue here is a conflict between two versions of `com.android.support:exifinterface`
But if there is different case where the changes to the manifest would fix the issue, then I can leave in also the manifest lines

Reply:
I'm not sure if they are necessary as they didn't really fix the issue, at least for me.
The issue here is a conflict between two versions of `com.android.support:exifinterface`
But if there is different case where the changes to the manifest would fix the issue, then I can leave in also the manifest lines

Reply:
I think they are 2 different kind of problems

and we should provide solutions for both

Reply:
Ok, then I will leave also those lines, I will also add this `exclude group: "com.google.android.gms"` line which is necessary if you use react-native-firebase

Reply:
tks